requirements/requirements-python3.8.txt

```
@@ -80,7 +80,7 @@ cached-property==1.5.1
 cachetools==4.1.0
 cassandra-driver==3.20.2
 cattrs==1.0.0
-celery==4.4.5
+celery==4.4.6
```

```suggestion
celery==4.4.7
```
@auvipy should we try?